js/index.js

```diff
@@ -101,3 +101,96 @@ modalTriggers.forEach((trigger) => {
     });
   });
 });
+
+function popup(data) {
+  if (
```

Comment:
Same for this create separate Boolean variables

Reply:
Done with changes. can you please check latest commit message